src/main/kotlin/no/nav/tiltakspenger/datadeling/domene/Barnetillegg.kt

```diff
@@ -0,0 +1,12 @@
+package no.nav.tiltakspenger.datadeling.domene
+
+import no.nav.tiltakspenger.libs.periodisering.Periode
+
+data class Barnetillegg(
```

Comment:
Siden vi kun skal motta, lagre og dele denne informasjonen velger jeg å ikke konvertere til periodisering. Hvis man skulle gjort noe logikk med denne burde man hatt en objekt for dette med periodisering også.